code/__DEFINES/networks.dm

```diff
@@ -37,8 +37,6 @@
 #define __NETWORK_CIRCUITS "CIRCUITS"
 
 /// Various combined subnetworks
-#define NETWORK_DOOR_REMOTES NETWORK_NAME_COMBINE(__NETWORK_DOORS, __NETWORK_REMOTES)
-#define NETWORK_DOOR_AIRLOCKS NETWORK_NAME_COMBINE(__NETWORK_DOORS, __NETWORK_AIRLOCKS)
```

Comment:
You should also remove ``__NETWORK_DOORS``, ``__NETWORK_REMOTES`` and ``__NETWORK_AIRLOCKS``, and from the unit test that uses them